contracts/contracts/coordination/Coordinator.sol

```diff
@@ -138,8 +138,43 @@ contract Coordinator is Initializable, AccessControlDefaultAdminRulesUpgradeable
         }
     }
 
-    function rituals(uint32 ritualId) public view returns (Ritual memory) {
-        return storageRitual(ritualId);
+    function rituals(
+        uint256 ritualId // uint256 for backward compatibility
+    )
+        external
+        view
+        returns (
+            address initiator,
+            uint32 initTimestamp,
+            uint32 endTimestamp,
+            uint16 totalTranscripts,
+            uint16 totalAggregations,
+            //
+            address authority,
+            uint16 dkgSize,
+            uint16 threshold,
+            bool aggregationMismatch,
+            //
+            IEncryptionAuthorizer accessController,
+            BLS12381.G1Point memory publicKey,
+            bytes memory aggregatedTranscript,
+            IFeeModel feeModel
```

Comment:
What about we create a dummy struct for this? Something like `RitualView` or along those lines.

Reply:
I doubt it will look any better, but if you want I can do that

Reply:
If you don't think it's worth it, then it's fine, let's keep it this way